MapboxDirections/Match/MBMatchOptions.swift

```diff
@@ -0,0 +1,82 @@
+import Foundation
+
+@objc(MBMatchOptions)
+open class MatchOptions: RouteOptions {
+    public init(tracePoints: [TracePoint], profileIdentifier: MBDirectionsProfileIdentifier?, resample: Bool?, timestamps: [Date]?) {
```

Comment:
@1ec5 do you think it makes sense to init this with tracePoints? TracePoints seem more like the result from a matching request.

Reply:
I think CLLocation would be a better choice, because a few of the properties on Tracepoint don’t make sense as part of a request.